SCADA-LTS/Scada-LTS
# Feature/#1936 purge data values limit

Purge data with values limit functionality added:
- new select with Purge strategy in Point properties in Logging properties section
- new text field (Values limit) in Point properties in Logging properties section
- new control (Values limit for purge) in System settings in Other settings section
- helps updated with new properties
- migration added
- lower limit in purging with period increased to 2 - at least two values will stay in database when purging is performed

Test scenario:
1. In System settings change **Values limit for purge** to 10
2. Create Virtual DS - update period - 1 s, enabled
3. Create two datapoints: dp1, dp2 - binary, change type - alternate, enabled
4. Go to Point properties of dp1 and select option **Values limit** in Logging properties section and save
5. Go to System settings and in **System information** next to **Database size** click **Refresh** button
6. Information about number of point values of our two datapoints should be visible
7. Click button **Purge now using point settings**
8. Number of values of dp1 should change to 10
